recipes/binutils/all/test_package/conanfile.py

```diff
@@ -35,16 +57,35 @@ def _test_package_assembly_source(self):
         return os.path.join(self.source_folder, f"{self._target_os}-{part_arch}.s")
 
     def _append_gnu_triplet(self, exe):
-        return f"{self.deps_user_info['binutils'].prefix}{exe}"
+        return f"{self._binutils_data['prefix']}{exe}"
+
+    def _run(self, command, output=None):
+        try: # v1 version
```

Comment:
Please, don't do that. Use separated folder like illustrated in CCI templates: https://github.com/conan-io/conan-center-index/tree/master/docs/package_templates/prebuilt_tool_package/all

Reply:
Thank you, will do